samples/net/openthread/coap_client/README.rst

```diff
@@ -0,0 +1,77 @@
+.. _coap_client_sample:
+
+openthread: coap_client
+##################
+
+The openthread coap_client sample demonstrates interactions between nodes performing different Thread roles with the use of OpenThread and CoAP protocol.
+
+Overview
+********
+
+The client node can control the state of BSP_LED_3 on server nodes. It can turn on the LED either on every server node in the network with a multicast message, or on a single specific server node that is paired with the client node.
+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF5340DK|
```

Comment:
inny dk?

Reply:
swoją drogą może fajnie było by posprawdzać na innych jeśli mamy.

Reply:
Osobiście mam 33, ją sprawdzę.

Reply:
lol. used polish here, my bad. sry:P
If 33 has all the buttons needed that would be awesome to try it and support both boards. If you're in the office and find any other board and have some time to spare you could test it. however this is not the priority right now. We are preparing this for an experimental release so one board should be enough.